pertpy/data/_datasets.py

```diff
@@ -141,6 +141,58 @@ def sciplex3_raw() -> AnnData:  # pragma: no cover
     return adata
 
 
+def haber_counts() -> AnnData:  # pragma: no cover
```

Comment:
I think that we already have this? https://github.com/theislab/pertpy/blob/development/pertpy/data/_datasets.py#L415

Reply:
The existing one is cell-level original dataset. And the newly added is aggregated count data

Reply:
As discussed IRL I'd remove the counts dataset and always work with the regions one in the tutorials to demonstrate how to get from a normal cell level one to counts.

Reply:
Haber counts data should not be necessary anymore. I'll let you guys finally remove it though before I forget sth somewhere